### ISSUE TITLE
Multi-select + drag

### ISSUE THREAD

**Issue by dsoiM** [jerosoler/Drawflow#50]
Hi, great project, good features and works nicely! Also quite customizable and it provides is HTML nodes, which other libraries really do not.
Have you considered adding multi-select + multi-drag feature?

**Comment by jerosoler** [comment]
Hello

Thanks! 😉

Yes I have considered it.

I think the best option would be CTRL + Click to nodes and drag.

Roger that. I don't promise this soon.

**Comment by dsoiM** [comment]
Yes agreed best and easiest option for implementing would be CTRL+click. Considering user experience could dragbox to select multiple be better? Lets say you want to move group of 30 nodes slightly to left, thats  gonna be a lot of clicking with CTRL+click

**Comment by jerosoler** [comment]
If you're right, a dragbox is better.
I'll take a look at it.

**Comment by dsoiM** [comment]
Combination of both is probably the best :-)

**Comment by danielmalencar** [comment]
This is very important. Without this feature it is not feasible to use the system

**Comment by devmondo** [comment]
such a beautiful and lite lib! thanks a lot for the great work :)

but i agree with @danielmalencar, without these capabilities it is really falling short :(

and having ability to select/unselect by Ctrl or lasso drag is crucial for node based UI

something like this might help?

https://simonwep.github.io/selection/

https://github.com/ThibaultJanBeyer/DragSelect

**Comment by Lazybone** [comment]
Is there a chance that a multiselect like DragSelect will be implemented?

**Comment by Lazybone** [comment]
is there any hope that Multi-Select will come?

**Comment by jerosoler** [comment]
There's always hope.

I'll start working on it. I don't promise to have it soon. It is complex.
But it is a feature that I would like to have.

**Comment by dearsina** [comment]
@jerosoler I just wanted to thank you for your work, this library is a favourite of mine. The multi-select feature would be great, and very valuable. I look forward to seeing the update!

**Comment by jerosoler** [comment]
Possible solution without drag, only shiftkey. 
- https://github.com/jerosoler/Drawflow/issues/322#issuecomment-1133036432
